LadderManager.cpp

```diff
@@ -927,7 +929,7 @@ void LadderManager::LoadAgents()
 					std::cerr << "Unable to parse file name for bot " << NewBot.BotName << std::endl;
 					continue;
 				}
-				if (!sc2::DoesFileExist(NewBot.RootPath + NewBot.FileName))
+				if (!(CanFindFileInEnvironment(NewBot.RootPath + NewBot.FileName)))
```

Comment:
Is there a good reason for this change?  It seems like the sc2 version would be better for maintaining compatibility across platforms

Reply:
This is something I debated over a bit.
I agree - I didn't want to stray away from using the sc2 version because of what you said as well as it being code that isn't our responsibility to maintain.
Unfortunately `sc2::DoesFileExist()` will not find a file/executable in the `PATH`. This means it can fail despite the executable reference being valid and therefore broke the test. I figured being able to run executables from the path was a useful feature, so I implemented that in order to fix the tests.